src/components/cron.ts

```diff
@@ -0,0 +1,22 @@
+import { updateSuggestionCron } from './suggestions';
+import { TextChannel } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+import { CronJob } from 'cron';
+
+const NOTIF_CHANNEL_ID_OLD: string = process.env.NOTIF_CHANNEL_ID_OLD || '.';
+
+// Checks for new suggestions every min
+export const createSuggestionCron = (client: CommandoClient): CronJob =>
+  new CronJob('0 */1 * * * *', async function () {
+    const isSuggestionsChanged = await updateSuggestionCron();
```

Comment:
instead of marking all the created suggestions as pending here, we should probably first query for all the created suggestions, then notify/show these suggestions to the channel in an embed, then call a function to update these suggestions as pending